pkg/test/ginkgo/test_suite.go

```diff
@@ -154,17 +154,15 @@ func (s *TestSuite) Filter(tests []*testCase) []*testCase {
 }
 
 func (s *TestSuite) AddRequiredMatchFunc(matchFn TestMatchFunc) {
-	if matchFn == nil {
-		return
-	}
 	if s.Matches == nil {
 		s.Matches = matchFn
-		return
 	}
 
-	originalMatchFn := s.Matches
-	s.Matches = func(name string) bool {
-		return originalMatchFn(name) && matchFn(name)
+	if matchFn != nil {
```

Comment:
am I missing the behavior change?  golang standard is lowest nesting. Is this refactor simply for taste that doesn't conform to golang/kube standard?

Reply:
I'm simply restoring it the way it was before all your changes:
https://github.com/openshift/origin/blob/0a89c44504306998fba408e83dd521a7e769c2f2/pkg/test/ginkgo/cmd_runsuite.go#L168-L173

The lack of `originalMatchFn` matching is required to properly filter the tests for the suite. It doesn't work if you early exit on the matchFn being nil, because it's always nil. Hence you're never skipping any tests.

I'm happy to take on whatever golang standard format you like, as long as it continues to skip the tests properly.

Reply:
This conditional looks unnecessary once https://github.com/openshift/origin/pull/28223/files#r1317293369 is fixed

Reply:
reviewing with whitespace off appears to demonstrate.

Reply:
Skips are definitely working (that's how serial tests work).  Perhaps a test to demonstrate?